napari/_vispy/_tests/test_utils.py

```diff
@@ -1,19 +1,33 @@
 import numpy as np
+import pytest
 from vispy.util.quaternion import Quaternion
 
 from napari._vispy.quaternion import quaternion2euler
 
+# Euler angles to be tested, in degrees
+angles = [[12, 53, 92], [180, -90, 0], [16, 90, 0]]
 
-def test_quaternion2euler(qtbot):
+# Prepare for input and add corresponding values in radians
+angles_param = [(x, True) for x in angles]
+angles_param.extend([(x, False) for x in np.radians(angles)])
+
+
+@pytest.mark.parametrize('angles_param', angles_param)
+def test_quaternion2euler(angles_param):
```

Comment:
```suggestion
@pytest.mark.parametrize('angles,degrees', angles_param)
def test_quaternion2euler(angles, degress):
```
```suggestion
@pytest.mark.parametrize('angles_param', angles_param)
def test_quaternion2euler(angles_param):
```

If you look here https://docs.pytest.org/en/stable/parametrize.html you can actually unpack the `angles_param` before passing to the test function. This will make everything a little easier to read